Pod/Classes/SeafConnection.m

```diff
@@ -1130,12 +1130,12 @@ - (void)pickPhotosForUpload
         file.retryable = false;
         file.autoSync = true;
         file.overwrite = true;
+        file.starred = photoAsset.isFavourite;
```

Comment:
Thanks. Maybe set starred in `- (void)setPHAsset:(PHAsset *)asset url:(NSURL *)url` will be better.

Reply:
Updated accordingly.